src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -128,6 +133,25 @@ public async Task<TSConfigResponse> TsConfigFiles(string tsConfigFilePath, Cance
             return JsonConvert.DeserializeObject<TSConfigResponse>(responseString);
         }
 
+        public async Task<AnalysisResponse> AnalyzeTs(string filePath, string tsConfigFilePath, CancellationToken cancellationToken)
+        {
+            var analysisRequest = new AnalysisRequest
+            {
+                FilePath = filePath,
+                IgnoreHeaderComments = true,
+                TSConfigFilePaths = new[] {tsConfigFilePath}
+            };
+
+            var responseString = await httpWrapper.PostAsync("analyze-ts", analysisRequest, cancellationToken);
+
+            if (string.IsNullOrEmpty(responseString))
+            {
+                throw new InvalidOperationException(Resources.ERR_InvalidResponse);
+            }
+
+            return JsonConvert.DeserializeObject<AnalysisResponse>(responseString);
+        }
```

Comment:
Apart from the `tsconfigFilePath`, this method is identical to the `AnalyzeJs` method one, isn't it? If so, it seems like a good candidate for factoring out most of the behaviour into a helper method.